conversations.yaml

```diff
@@ -71,7 +73,9 @@ paths:
         - Channel Adapter Services
       description:
         Returns a customer's profile data for use in Medallia
-        Conversations' reporting interfaces.
+        Conversations' reporting interfaces. There are no required fields. 
```

Comment:
Change `There are no required fields.` to `All fields returned by the adapter are optional.`  In client-facing communications, always avoid "negative" words like "no".